zipkin-anormdb/build.gradle

```diff
@@ -0,0 +1,16 @@
+// Database drivers
+ext.anormDriverDependencies = [
```

Comment:
It's just DRY; same reason for all the other helpers using `mkDepGen`. The map is needed because we need different versions for different packages.

What would your implementation look like?